api/opencl_runtime_layer.asciidoc

```diff
@@ -1996,6 +1996,39 @@ The following restrictions apply when mipmapped images are created with
     images, depth images or multi-sampled (i.e. msaa) images.
 endif::cl_khr_mipmap_image[]
 
+ifdef::cl_ext_image_tiling_control[]
+*Tiling Control*
+
+{CL_MEM_IMAGE_TILING_EXT_anchor} can be passed as part of the _properties_ parameter
+to {clCreateImageWithProperties} to control the tiling used for the image being
+created. The following values are accepted:
+
+  * {CL_IMAGE_TILING_LINEAR_EXT_anchor}, which is the default value used if
+    {CL_MEM_IMAGE_TILING_EXT} is not specified in the _properties_. Image data
```

Comment:
Discussed in the 2025/01/28 memory TSG: agreement to not require linear tiling by default but let the implementation pick the tiling.

Reply:
These look good

Reply:
I have now reflected the agreed change in the draft specification.